--- libccid_Info.plist_org	2023-10-17 10:50:56.030148081 +0200
+++ libccid_Info.plist_test	2024-01-16 15:11:54.798891142 +0100
@@ -462,6 +462,8 @@
 		<string>0x2D25</string>
 		<string>0x2C97</string>
 		<string>0x2C97</string>
+		<string>0x2C97</string>
+		<string>0x2C97</string>
+		<string>0x2C97</string>
 		<string>0x17EF</string>
 		<string>0x17EF</string>
 		<string>0x17EF</string>
@@ -1005,8 +1007,10 @@
 		<string>0x43A9</string>
 		<string>0x0000</string>
 		<string>0x0001</string>
-		<string>0x0001</string>
-		<string>0x0004</string>
+		<string>0x1009</string>
+		<string>0x4009</string>
+		<string>0x5009</string>
+		<string>0x6009</string>
+		<string>0x7009</string>
 		<string>0x6007</string>
 		<string>0x6055</string>
 		<string>0x6111</string>
@@ -1552,6 +1556,8 @@
 		<string>KRONEGGER Micro Core Platform</string>
 		<string>Ledger Nano S</string>
 		<string>Ledger Nano X</string>
+		<string>Ledger Nano S Plus</string>
+		<string>Ledger Stax</string>
+		<string>Ledger Flex</string>
 		<string>Lenovo Lenovo USB Smartcard Keyboard</string>
 		<string>Lenovo Lenovo USB Smartcard Keyboard</string>
 		<string>Lenovo Lenovo Smartcard Wired Keyboard II</string>
